src/plugins/gitresolver/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+include (LibAddMacros)
+#if (DEPENDENCY_PHASE)
+    #    find_package(LibGit2)
```

Comment:
Seems like FindLibGit2.cmake is not installed by upstream? What upstream does seems quite useless anyway, they use FIND_PATH and  FIND_LIBRARY with empty variables as hint:
https://github.com/libgit2/libgit2-backends/commits/master/CMake/FindLibgit2.cmake

So I ended up writing something minimal myself:
9f2ca5a3722d08772e19684bddfc18a9e8663b37

Please rebase, than your code should work.